src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -2941,6 +2941,9 @@
   <data name="NotSupported_OutputStreamUsingTypeBuilder" xml:space="preserve">
     <value>Output streams do not support TypeBuilders.</value>
   </data>
+  <data name="NotSupported_Overlapped" xml:space="preserve">
```

Comment:
There is an existing string `PlatformNotSupported_OverlappedIO`. Unify them?

Reply:
I wasn't sure since it seems to be used slightly differently:
- PlatformNotSupported_OverlappedIO says `This API is specific to the way in which Windows handles asynchronous I/O, and is not supported on this platform.` and is used here: https://github.com/dotnet/runtime/blob/1e09e983d6959e22f9884e6060e0245a0040bfd1/src/coreclr/src/System.Private.CoreLib/src/System/Threading/ClrThreadPoolBoundHandle.Unix.cs#L18
- NotSupported_Overlapped says `This API is not supported on this platform.` and is used here: https://github.com/dotnet/runtime/blob/1e09e983d6959e22f9884e6060e0245a0040bfd1/src/libraries/System.Private.CoreLib/src/System/Threading/ThreadPoolBoundHandle.PlatformNotSupported.cs#L29

I'll keep them separate for now :)

Reply:
Both of these are supposed to mean and say the same thing. The reason why the message differs is that one came from the .NET Native/CoreRT fork and other one from the CoreCLR fork.

Reply:
Ok, I consolidated on PlatformNotSupported_OverlappedIO.

Reply:
Perfect. Thanks!